microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -1,376 +1,556 @@
+* Providing or enabling access to content or services that are illegal under applicable local law.
 
-### 2.12 Age Ratings
 
-#### 2.12.1 Mature Content
+<!-- ------------------------------ -->
+#### 2.12 Age ratings
 
-When you submit your extension to [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), you must indicate whether your extension displays content that should be marked "Mature".  When determining the rating for your extension, consider all the content in your app, including user generated content and ads, and to the content that your extension links.  If you indicate that your extension doesn't contain any "Mature" content, you are responsible for maintaining the accuracy of this rating.
 
-Regardless of the rating given to your extension, it must still adhere to all the content requirements of Microsoft Edge Add-ons Developer policies.
+###### 2.12.1 Mature content
 
-#### 2.12.2 Ratings Change
+When you submit your extension to Partner Center, you must indicate whether your extension displays content that should be marked "Mature".  When determining the rating for your extension, consider all the content in your app, including user generated content and ads, and to the content that your extension links.  If you indicate that your extension doesn't contain any "Mature" content, you are responsible for maintaining the accuracy of this rating.
+
+Regardless of the rating given to your extension, it must still adhere to all the content requirements of the Microsoft Edge Add-ons store developer policies.
+
+
+###### 2.12.2 Ratings change
 
 If your extension provides content (such as user-generated, retail, or other web-based content) that might be appropriate for a higher age rating than the assigned rating, you must require users to opt into receiving such content by using a content filter or by signing in with a pre-existing account.
 
-### 2.13 Videos
 
-If you submit a promotional video in the listing, it should follow all the content guidelines mentioned in this policy.  If you choose to provide a YouTube link, you must ensure that advertisements are turned off for the specific videos that you want to embed.  For more information on turning on or off ads on YouTube, see [Set your default ad formats](https://support.google.com/youtube/answer/2531367?ref_topic=7072227) and [Ads on embedded videos](https://support.google.com/youtube/answer/132596).
+<!-- ------------------------------ -->
+#### 2.13 Videos
 
+If you submit a promotional video in the listing, then it should strictly adhere to our content guidelines.  If you choose to provide a YouTube link, you must ensure that advertisements are turned off for the specific videos that you want to embed.  For more information on turning on or off ads on YouTube, see [YouTube advertising formats](https://support.google.com/youtube/answer/2531367?ref_topic=7072227) and [Ads on embedded videos](https://support.google.com/youtube/answer/132596).
```

Comment:
```suggestion
If you submit a promotional video in the listing, the promotional video must strictly adhere to the Developer policies for the Microsoft Edge Add-ons store.  If you choose to provide a YouTube link, you must ensure that advertisements are turned off for the specific videos that you want to embed.  For more information on turning on or off ads on YouTube, see [YouTube advertising formats](https://support.google.com/youtube/answer/2531367?ref_topic=7072227) and [Ads on embedded videos](https://support.google.com/youtube/answer/132596).
```
"our" is bad, too personal/possessive.  reword to avoid "our"
@Bhuvana-Priya 
"it should strictly adhere to our content guidelines."
Solve by being specific.
Alts:
* it should strictly adhere to the Developer policies for the Microsoft Edge Add-ons store.
* it should strictly adhere to the content guidelines in documents x, y, and z.

Reply:
If you submit a promotional video in the listing, then it should strictly adhere to these content policies.